Junk4Dollars-appUITests/Junk4Dollars_appUITests.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Junk4Dollars_appUITests.swift
+//  Junk4Dollars-appUITests
+//
+//  Created by John Riccardi on 10/28/19.
+//  Copyright © 2019 John Riccardi. All rights reserved.
+//
+
+import XCTest
+
+class Junk4Dollars_appUITests: XCTestCase {
+    func testHelloDefault() {
+        let app = XCUIApplication()
+        app.launch()
+        app.buttons["hello_btn"].tap()
+        let helloLabel = app.staticTexts["output_label"]
+        XCTAssert(helloLabel.exists)
+    }
+    
+    func testHelloName() {
+        let app = XCUIApplication()
+        app.launch()
+        let textInput = app.textFields["text_input"]
+        textInput.tap()
+        textInput.typeText("John")
+        app.buttons["hello_btn"].tap()
+        let helloJohnLabel = app.staticTexts["Hello, John!"]
+        XCTAssert(helloJohnLabel.exists)
+    }
+    
+    func testResetButton() {
+        let app = XCUIApplication()
+        app.launch()
+        let textInput = app.textFields["text_input"]
+        textInput.tap()
+        textInput.typeText("John")
+        app.buttons["hello_btn"].tap()
+        app.buttons["reset_btn"].tap()
+        app.buttons["hello_btn"].tap()
+        XCTAssert(app.staticTexts["Hello!"].exists)
+    }
+
+    func testLaunchPerformance() {
```

Comment:
What exactly does this do? Does it just log the metrics?

Reply:
Yeah, it opens and closes the app 6 times and logs the metrics to make sure the load times don't start increasing too much.